.github/workflows/e2e.yaml

```diff
@@ -15,70 +15,73 @@
+        required: true
+        type: string
+      golang_version:
+        required: true
+        type: string
+    secrets:
+      AWS_ACCESS_KEY_ID:
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        required: true
+      AWS_SSH_KEY:
+        required: true
+      SLACK_BOT_TOKEN:
+        required: true
+      SLACK_CHANNEL_ID:
+        required: true
 
 jobs:
   e2e-tests:
     runs-on: linux-amd64-cpu4
-    if: ${{ github.event.workflow_run.conclusion == 'success' }} && ${{ github.event.workflow_run.event == 'push' }}
     steps:
-      - name: Check out code
-        uses: actions/checkout@v4
+    - name: Check out code
+      uses: actions/checkout@v4
 
-      - name: Calculate build vars
-        id: vars
-        run: |
-          echo "COMMIT_SHORT_SHA=${GITHUB_SHA:0:8}" >> $GITHUB_ENV
-          GOLANG_VERSION=$(./hack/golang-version.sh)
-          echo "GOLANG_VERSION=${GOLANG_VERSION##GOLANG_VERSION := }" >> $GITHUB_ENV
+    - name: Install Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ inputs.golang_version }}
 
-      - name: Install Go
-        uses: actions/setup-go@v5
-        with:
-          go-version: ${{ env.GOLANG_VERSION }}
+    - name: Set up Holodeck
+      uses: NVIDIA/holodeck@v0.2.5
+      with:
+        aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws_ssh_key: ${{ secrets.AWS_SSH_KEY }}
+        holodeck_config: "tests/e2e/infra/aws.yaml"
 
-      - name: Set up Holodeck
-        uses: NVIDIA/holodeck@v0.2.5
-        with:
-          aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws_ssh_key: ${{ secrets.AWS_SSH_KEY }}
-          holodeck_config: "tests/e2e/infra/aws.yaml"
+    - name: Run e2e tests
+      env:
+        KUBECONFIG: ${{ github.workspace }}/kubeconfig
+        E2E_IMAGE_REPO: ghcr.io/nvidia/k8s-device-plugin
+        E2E_IMAGE_TAG: ${{ inputs.version }}-ubi9
+        LOG_ARTIFACTS: ${{ github.workspace }}/e2e_logs
+      run: |
+        make test-e2e
 
-      - name: Run e2e tests
-        env:
-          KUBECONFIG: ${{ github.workspace }}/kubeconfig
-          E2E_IMAGE_REPO: ghcr.io/nvidia/k8s-device-plugin
-          E2E_IMAGE_TAG: ${COMMIT_SHORT_SHA}-ubi9
-          LOG_ARTIFACTS: ${{ github.workspace }}/e2e_logs
-        run: |
-          make test-e2e
+    - name: Archive test logs
+      if: ${{ failure() }}
+      uses: actions/upload-artifact@v4
+      with:
+        name: e2e-test-logs
+        path: ./e2e_logs/
+        retention-days: 15
 
-      - name: Archive test logs
-        if: ${{ failure() }}
-        uses: actions/upload-artifact@v4
-        with:
-          name: e2e-test-logs
-          path: ./e2e_logs/
-          retention-days: 15
+    - name: Send Slack alert notification
+      id: slack
+      if: false
+      uses: slackapi/slack-github-action@v2.0.0
```

Comment:
@ArangoGutierrez the upgrade to v2.0.0 has broken this step. I believe the API has changed.

Reply:
Let me check and take action for our repos

Reply:
fixed